examples/kernel/variables.py

```diff
@@ -1,14 +1,15 @@
 import pyomo.kernel as pmo
+import pyomo.environ as pe
```

Comment:
I'm hoping to keep pyomo.kernel as self contained as possible until the two modeling layers are compatible. Is this import necessary?

Reply:
Unfortunately, for this example., yes: `Reals` is a proper Pyomo Set.  The alternative was to change all the `Reals` in the example to `RealSet` from kernel.